vanagon.gemspec

```diff
@@ -12,6 +12,8 @@ Gem::Specification.new do |gem|
   gem.authors  = ['Puppet Labs']
   gem.email    = 'info@puppetlabs.com'
   gem.homepage = 'http://github.com/puppetlabs/vanagon'
+  gem.specification_version = 3
+  gem.required_ruby_version = '~> 2.1.0'
```

Comment:
@underscorgan I don't expect it to break on a 2.x release. it works fine on 2.2, but I'm open to all of the possibilities.